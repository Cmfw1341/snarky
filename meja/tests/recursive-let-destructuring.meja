let rec (x, _) = (15, ());
