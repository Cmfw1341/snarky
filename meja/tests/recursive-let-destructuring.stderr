File "tests/recursive-let-destructuring.meja", line 1, characters 9-10:
Error: Cannot bind x in this recursive let:
it must have a functional type.
